lib/mark.rb

```diff
@@ -0,0 +1,8 @@
+class Mark
+  X = "X"
+  O = "O"
+
+  def Mark.is_a_mark?(mark)
```

Comment:
For class methods using `self` is preferred over the name of the class since it wont be affected if you decide to change the name of the class.